### PR TITLE
Add PyPrind recipe

### PR DESCRIPTION
**Pull request by cmutel**

### CODE REVIEW

**Comment by synapticarbors** [comment]
Looks good to me. The only thing I wasn't sure about was whether `psutil` is needed as a build dependency.

**Comment by cmutel** [comment]
Thanks - I thought it was needed because `setup.py` imported the package to get the version number, but it seems to build fine without it.

File: recipes/pyprind/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - psutil >=3.2.0
+
+test:
+  imports:
+    - pyprind
+
+about:
+  home: https://github.com/rasbt/pyprind
+  license: BSD-3-Clause
```
Comment (jakirkham) [inline recipes/pyprind/meta.yaml]: Could add `license_family: BSD`. Though it is optional.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**